var/spack/repos/builtin/packages/py-bluepyefe/package.py

```diff
@@ -11,6 +11,9 @@ class PyBluepyefe(PythonPackage):
 
     homepage = "https://github.com/BlueBrain/BluePyEfe"
     url = "https://pypi.io/packages/source/b/bluepyefe/bluepyefe-0.3.13.tar.gz"
+    git = "https://github.com/BlueBrain/BluePyEfe.git"
+
+    version('BPE2', branch='BPE2')
```

Comment:
fine with me. just a note that a dated version with a commit may be better (as there will be no reference to the commit used in spack like this).

Reply:
yes, I know this is not clean, but it should not be used in production, only part of the main BluePyEmodel will be used, which does not require this code. At some point this branch will be the master one, and we'll clean this up.

Reply:
Did this "point" arrived? Just wondering what happened to the branch?

Reply:
Yes, the point arrived. The branch became the new master.

Reply:
Could you please update the spack package as build is failing because of the missing branch: https://bbpgitlab.epfl.ch/hpc/spack/-/jobs/562713#L2506
Thanks!